### PR TITLE
Adjusted the MQTT API by always using the IMqttClient Interface 

### PR DESCRIPTION
**Pull request by Frank-Wiebeler**
…instead of sometimes using IMqttClient and sometimes using MqttClient as Parameter.

### CODE REVIEW

File: agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/SetSubscriptionService.java [hunk 1]
```diff
@@ -2,7 +2,5 @@
 
 import com.dke.data.agrirouter.api.service.parameters.SetSubscriptionParameters;
 
-/**
- * Service to set the subscriptions for an endpoint.
- */
+/** Service to set the subscriptions for an endpoint. */
```
Comment (saschadoemer) [inline agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/SetSubscriptionService.java]: Why? Is this your formatter?

File: agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/MessageEncoder.java [hunk 2]
```diff
@@ -299,6 +299,7 @@ default EncodedMessage encode(
 
   /**
    * Get the service to encode messages.
+   *
```
Comment (saschadoemer) [inline agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/MessageEncoder.java]: Why? Is this your formatter?

File: agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/SetSubscriptionService.java (hunk 1 above)
Comment (Frank-Wiebeler) [reply agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/SetSubscriptionService.java]: No, it's the mvn formatter and this is the correct formatting.
It used to be wrong and was fixed in #86 .
As this branch origins from an older version of develop, it's shown as a change.
Actually, it's the same as in the current develop branch.

File: agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/MessageEncoder.java (hunk 2 above)
Comment (Frank-Wiebeler) [reply agrirouter-api-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/MessageEncoder.java]: No, it's the mvn formatter and this is the correct formatting.
It used to be wrong and was fixed in #86 .
As this branch origins from an older version of develop, it's shown as a change.
Actually, it's the same as in the current develop branch.

**Pull request merged**